### PR TITLE
#31 Created landing page

### PR DESCRIPTION
**Pull request by Krymets**
Created landing page

### CODE REVIEW

File: FrontEnd/main/src/components/landing-page/ellipses/Ellipses.js [hunk 1]
```diff
@@ -0,0 +1,40 @@
+import './Ellipses.css';
+
+export const GreenEllipses = () => {
+  return (
+    <div className="ellipses-main">
+      <div className="ellipses-main__green"/>
```
Comment (maxvonlancaster) [inline FrontEnd/main/src/components/landing-page/ellipses/Ellipses.js:6]: Would probably be better to pass type of ellipses as props - and have one generic component Ellipses

File: FrontEnd/main/src/components/landing-page/divider/Divider.js [hunk 2]
```diff
@@ -0,0 +1,7 @@
+import './Divider.css';
+
+export const MenuDivider = () => {
+  return (
+      <div className="divider"/>
```
Comment (maxvonlancaster) [inline FrontEnd/main/src/components/landing-page/divider/Divider.js:5]: Do we really need a separate component for this?

**Review by maxvonlancaster — APPROVED** [review]

File: FrontEnd/main/src/components/landing-page/partners/Partners.js [hunk 3]
```diff
@@ -0,0 +1,33 @@
+import './Partners.css';
+
+const MainPartners = () => {
+    return (
+        <div className="partners-section">
+          <div className="partners-section__header">
+              <div className="partners-section__header-text">партнери</div>
```
Comment (maxvonlancaster) [inline FrontEnd/main/src/components/landing-page/partners/Partners.js:7]: For the future, having file LocalizedText.js (or some other name) with all this text, that is displayed for the user would be great

File: FrontEnd/main/src/components/landing-page/divider/Divider.js (hunk 2 above)
Comment (Krymets) [reply FrontEnd/main/src/components/landing-page/divider/Divider.js]: Divider component removed

File: FrontEnd/main/src/components/landing-page/partners/Partners.js (hunk 3 above)
Comment (Krymets) [reply FrontEnd/main/src/components/landing-page/partners/Partners.js:7]: Thanks

**Review by mehalyna — APPROVED** [review]

**Pull request merged**